packages/stores/types.ts

```diff
@@ -281,6 +282,19 @@ type TTradersHubStore = {
     selectRegion: (region: string) => void;
 };
 
+type TTradeStore = {
```

Comment:
@henry-deriv There is no point in adding partial types for trade-store. Updating this file is out of the scope of this PR, we don't have to add types to the stores, instead we need to add types to the components assuming that the store will be typed later outside of this PR. )

Reply:
I did it this way because it makes my life easier when we reuse same type in other files in the same PR. Instead of repeating the same type over and over again.